code/api/src/modules/user/resolvers.js

```diff
@@ -77,3 +77,47 @@ export async function remove(parentValue, { id }) {
 export async function getGenders() {
   return Object.values(params.user.gender)
 }
+
+// Update user
+export async function update(parentValue, { id, style }) {
+  let styleArray = style.split(', ')
+  let styleCount = {}
+
+  styleArray.forEach((val) => {
+    if (!styleCount[val]) {
+      styleCount[val] = 1
+    }
+    else {
+      styleCount[val]++
+    }
+  })
+
+  let styleKeys = Object.keys(styleCount);
+
+  let styleKeysSorted = styleKeys.sort(function (a, b) { return styleCount[a] - styleCount[b] }).reverse();
+
+  let firstStyle = "";
+  let secondStyle = "";
+  let thirdStyle = "";
+
+  if (styleKeysSorted.length > 0) {
+    firstStyle = styleKeysSorted[0]
+  }
+
+  if (styleKeysSorted.length > 1) {
+    secondStyle = ` but ${styleKeysSorted[1]}`
+  }
+
+  if (styleKeysSorted.length > 2) {
```

Comment:
allows a style to be generated in the event that only 1 or 2 style types were selected